### DISCUSSION TITLE
How do you adjust mouse input based on a CRT-style shader?

### DISCUSSION THREAD

**Discussion by Thraka** [MonoGame/MonoGame#8264] [Graphics]
I have no idea where to begin with adjusting mouse hit (2d) detection based on how a CRT shader bends the screen. I'm using the old crt-lottes-mg.fx from https://community.monogame.net/t/anyone-have-crt-style-shaders/11023

Any help?

**Comment by ThomasFOG** [comment]
You might want to hide the mouse cursor and draw it yourself in your game so that it is drawn with the curvature. This way you don't have to reverse the coordinates and you can assume that the curvature doesn't exist.

**Comment by Thraka** [comment]
Thanks for the suggestion!
